brainstormforce/ultimate-addons-for-gutenberg
# Add Minimum Height option for slider

### Description
Added Minimum Height option for Slider 

### Types of changes
New feature (non-breaking change which adds functionality) 

### How has this been tested?
- Add a slider block on the page/post
- Select block, Navigation to General Settings Panel 
- You will see a new option for Minimum Height. 
- Adjust Height for desktop and other tablets
- Check if height is applied as per the option value. 

Note: Slider Height will be equal to Minimum Height + ( Top + Bottom padding )

### Checklist:
